index.js

```diff
@@ -51,13 +46,17 @@ module.exports = {
 		// ternary on multiline
 		'multiline-ternary': ['error', 'always-multiline'],
 		// force proper JSDocs
-		'valid-jsdoc': [2, {
+		'valid-jsdoc': ['error', {
 			prefer: {
 				return: 'returns'
 			},
 			requireReturn: false,
 			requireReturnDescription: false
 		}],
+		// disallow use of "var"
+		'no-var': 'error',
```

Comment:
we require es6, so let's enforce `let`/`const` instead of `var`

Reply:
Isn't it already what this does?

Reply:
Ah sorry! I misread :)